fs/ftpfs.py

```diff
@@ -440,11 +440,9 @@ def _manage_ftp(self):
     def ftp_url(self):
         # type: () -> Text
         """Get the FTP url this filesystem will open."""
-        url = (
-            "ftp://{}".format(self.host)
-            if self.port == 21
-            else "ftp://{}:{}".format(self.host, self.port)
-        )
+        _host_part = self.host if self.port == 21 else "{}:{}".format(self.host, self.port)
+        _user_part = None if self.user == 'anonymous' else "{}:{}@".format(self.user, self.passwd)
```

Comment:
Should this be ` _user_part = "" if ` ?

Reply:
Nice hint.
Yes, `= ""` will remove unnecessary cast of None to string. It's better! Thank you.